public/locales/en/common.json

```diff
@@ -13,5 +13,17 @@
     "subtitle": "Pray for",
     "read": "Read the entire article",
     "view": "View all topics"
+  },
+  "footer": {
+    "map": {
```

Comment:
I don't quite understand the need for "map" - what does it mean here? Do we need to be so nested?